bin/run_rajaperf.sh

```diff
@@ -48,9 +49,14 @@ build_targets="hip openmp"
 if [ "$2" == "build" ]; then
   # Begin configuration
   pushd $AOMP_REPOS_TEST/RAJAPerf
-  #cd $AOMP_REPOS_TEST/RAJAPerf
-  git reset --hard 13808861f5ce8348fbaa124e23fed535a2a1e1fb
+  git reset --hard 0e6972bb29a5d316241caf057eb5815a645e57e8
```

Comment:
Maybe make that SHA into a variable that can be set from the outside?

Reply:
Thanks for your review. :) I would prefer not to. The script is tested for this specific SHA and thus needs to be stored somewhere anyway. I think it's better to store it close to the use-site.